CRM/Custom/Import/Parser/Api.php

```diff
@@ -25,6 +25,7 @@ public static function getUserJobInfo(): array {
         'id' => 'custom_field_import',
         'name' => 'custom_field_import',
         'label' => ts('Multiple Value Custom Field Import'),
+        'entity' => 'Contact',
```

Comment:
From the API's POV, this isn't really the correct entity. But if it's working... fine.

Reply:
@colemanw  it is the correct entity from the import table's point of view though - ie the import table will stick a contact ID into the `entity_id` field (if it sticks anything in, but that is another story)